client/src/components/QandA/QandAContainer.jsx

```diff
@@ -83,13 +87,32 @@ function QandA({ product }) {
     })
   }
 
+  const handleMoreAnsweredQuestions = () => {
+    const howMany = questions.length - questionCount;
+    console.log(howMany);
+    switch(howMany) {
```

Comment:
I personally try to stray away from using switch as much as I can. If possible, try to refactor this later on.